client/vite.config.js

```diff
@@ -3,5 +3,15 @@ import react from '@vitejs/plugin-react'
 
 // https://vitejs.dev/config/
 export default defineConfig({
-  plugins: [react()],
+    plugins: [react()],
+    server: {
+        cors: true,
+        proxy: {
+            '/api': {
+                target: 'http://127.0.0.1:5000',
```

Comment:
You may need to add the target URL as an environment variable so that when the app is run in production it uses the correct URL in the different environments.

Reply:
Hi @otsembo . Isn't the server property ignored in a production environment when it comes to Vite? I had come across that somewhere but I could be wrong. I'll move it to an environment variable but I'm curious about whether this property counts in production.

Reply:
That is actually true. You are correct, it will be ignored in a production environment.